src/main/steps/common/components/common/renderSubFields.test.ts

```diff
@@ -22,17 +22,6 @@ describe('steps > common > components > common > renderSubFields', () => {
     //expect(subFields[radiosFieldName].type).toBe(radiosType);
   });
 
```

Comment:
The constants radiosFieldName, radiosType and values can be removed. And their reference in renderSubFields can be removed as well.